Ghidra/Features/Decompiler/src/main/java/ghidra/app/decompiler/DecompileOptions.java

```diff
@@ -784,7 +817,9 @@ public String getXML(DecompInterface iface) {
 		if (maxIntructionsPer != SUGGESTED_MAX_INSTRUCTIONS) {
 			appendOption(buf, "maxinstruction", Integer.toString(maxIntructionsPer), "", "");
 		}
+
 		appendOption(buf, "protoeval", protoEvalModel, "", "");
+		appendOption(buf, "indentationstyle", indentationStyle.getOptionString(), "", "");
```

Comment:
I confess that many of these calls to appendOption are contained within a 
```
if (option != OPTION_OPTIONDEFAULT) {
    ...
}
```

Clause, and I'm both not sure what this protects against, and whether the indentation style check should have a similar check.